src/main/java/command/AddCommand.java

```diff
@@ -0,0 +1,108 @@
+package command;
+
+import exception.InvalidDateFormat;
+import exception.NullDescription;
+import storage.Storage;
+import task.Deadline;
+import task.Task;
+import task.TaskList;
+import task.Todo;
+import task.Event;
+import ui.Ui;
+
+public class AddCommand extends Command {
+
+    private final boolean EXIT = false;
+    private String taskType;
+    private String taskInfo;
+
+    private void checkDescription(String description, String type) throws NullDescription {
+        int numOfCharacters =  type.length();
+        if (description.length() <= numOfCharacters || description.split(" ").length < 2) {
+            throw new NullDescription(type);
+        }
+    }
+
+    private void verifyDate(String dateInfo, String type) throws InvalidDateFormat {
+        switch(type) {
+            case "deadline":

```

Comment:
In order to better abide by the Java coding standard, that states that there is no indentation for case clauses, would it be better to remove the indentation here? (and similarly to other areas in your code if you have used switch, case)

Reply:
In general, I think the `addCommand.java` is pretty well done abiding by the coding standard mostly.  However, I believe for `switch` cases in this module are supposed to be aligned with the `switch`? (Same as Jingya's comment)

Reply:
Thanks for the catch, I'll amend them in the future commits!